roles/kiwix/tasks/enable-or-disable.yml

```diff
@@ -1,18 +1,20 @@
-- name: Enable & Restart 'kiwix-serve' systemd service
-  systemd:
-    name: kiwix-serve
-    daemon_reload: yes
-    enabled: yes
-    state: restarted
-  when: kiwix_enabled
-
 - name: Disable & Stop 'kiwix-serve' systemd service
   systemd:
     name: kiwix-serve
     enabled: no
     state: stopped
-  when: not kiwix_enabled
 
+- name: Update '{{ kiwix_library_xml }}'
+    command: /usr/bin/iiab-make-kiwix-lib
+  when: kiwix_enabled
+# the above starts the service
```

Comment:
The above starts which service?

The same `kiwix-serve.service` being started just below, or a different one?

Reply:
What role is this found in? kiwix_enabled implies the use of kiwix-serve.service, You are being pretty petty for a note that could just be removed,

Reply:
I was asking a sincere question.

I guess it's harmless that it (kiwix-serve.service) is "started" above and below, with these 2 stanzas in a row.